addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRuleUnitTest.java

```diff
@@ -57,13 +70,19 @@ protected void setUpMessages() {
     @Override
     @Test
     public void shouldAlertWhenRequestIsSuccessful() throws Exception {
+       // Override this to do nothing
+    }
+
+
+    @ParameterizedTest
+    @NullSource
+    @ValueSource(strings = {"application/octet-stream"})
+    void shouldAlertWhenRequestIsSuccessfulWithExpectedContent(String input) throws Exception {
```

Comment:
`input` → `contentType`

Reply:
resolved

Reply:
The code was not updated.

Reply:
hmm, I swear I updated this, but i just changed again and repushed.

Reply:
It is now, better push before saying so.